connectors/gemini/AGENTS.md

```
@@ -0,0 +1,31 @@
+<!-- MAG_MEMORY_START -->
+# MAG — Persistent Memory
+
+MAG provides persistent memory across sessions. Available commands:
+
+## Store a memory
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)